### PR TITLE
[feature] add blog use case Locxia

### PR DESCRIPTION
**Pull request by Ben8t**

### CODE REVIEW

File: content/blogs/2023-06-21-locxia-kestra.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+description: How Ntico Leverage Kestra Orchestrator for their data platform Locxia.
+date: 2023-06-22T18:00:00
+category: Solutions
+author:
+  name: Benoit Pimpaud
+  image: "bpimpaud"
+image: /blogs/2023-06-22-locxia-kestra.jpg
+---
+
+Ntico has designed [Locxia](https://www.locxia.com/), a powerful platform that gives businesses and local authorities a deeper understanding of their territories and residents thanks to satellite imagery. At the core of their solution, they leveraged Kestra for its features and efficient management of large-scale data, including ingesting, processing, and visualizing vast datasets.
```
Comment (anna-geller) [inline content/blogs/2023-06-21-locxia-kestra.md:12]:
```suggestion
[Locxia](https://www.locxia.com/), developed by Ntico, is a robust platform that gives businesses and local authorities a deeper understanding of their territories and residents thanks to satellite imagery. At the core of their solution lies Kestra, used for ingesting, processing, and visualizing vast datasets. 
```

File: content/blogs/2023-06-21-locxia-kestra.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+category: Solutions
+author:
+  name: Benoit Pimpaud
+  image: "bpimpaud"
+image: /blogs/2023-06-22-locxia-kestra.jpg
+---
+
+Ntico has designed [Locxia](https://www.locxia.com/), a powerful platform that gives businesses and local authorities a deeper understanding of their territories and residents thanks to satellite imagery. At the core of their solution, they leveraged Kestra for its features and efficient management of large-scale data, including ingesting, processing, and visualizing vast datasets. 
+
+Let's look at how Nitco creates and manages data pipelines for their digital solution with Kestra.
```
Comment (anna-geller) [inline content/blogs/2023-06-21-locxia-kestra.md:14]:
```suggestion
This post dives into how Nitco uses Kestra to create and manage data pipelines for its digital solution.
```

File: content/blogs/2023-06-21-locxia-kestra.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+
+## About Ntico
+
+[Ntico is a French consulting and software company](https://www.ntico.com/), specializing in developing data-driven applications and managing intricate projects.
```
Comment (anna-geller) [inline content/blogs/2023-06-21-locxia-kestra.md:19]:
```suggestion
[Ntico is a French consulting and software company](https://www.ntico.com/) specializing in data-driven applications. 
```

File: content/blogs/2023-06-21-locxia-kestra.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+Ntico has designed [Locxia](https://www.locxia.com/), a powerful platform that gives businesses and local authorities a deeper understanding of their territories and residents thanks to satellite imagery. At the core of their solution, they leveraged Kestra for its features and efficient management of large-scale data, including ingesting, processing, and visualizing vast datasets. 
+
+Let's look at how Nitco creates and manages data pipelines for their digital solution with Kestra.
+
+
+## About Ntico
+
+[Ntico is a French consulting and software company](https://www.ntico.com/), specializing in developing data-driven applications and managing intricate projects. 
+
+They provide custom consulting, data analysis, and software development services tailored to enhance operational efficiency.
```
Comment (anna-geller) [inline content/blogs/2023-06-21-locxia-kestra.md:21]:
```suggestion
They provide consulting, data analysis, and software development services tailored to enhance operational efficiency. 
```

File: content/blogs/2023-06-21-locxia-kestra.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+Additionally, Ntico offers operational services encompassing production, network infrastructure, cloud, and security solutions. Their expertise extends to advanced logistics solutions that maximize productivity and streamline supply chains.
+
+Ntico's unique offerings include Locxia, a data platform designed to develop comprehensive insights into territories and their inhabitants.
+
+
+## About Locxia
+
+![loxcia screenshot](blogs/2023-06-22-locxia-kestra/locxia-screenshot.png)
+
+Locxia is a data platform revolving around housing data. Developed by Ntico's data experts, it empowers businesses and local authorities to better understand their customers and territories. The platform provides access to over 250 value-added data sourced from satellite images and public data, all consolidated and accessible on a dedicated platform.
```
Comment (anna-geller) [inline content/blogs/2023-06-21-locxia-kestra.md:32]:
```suggestion
Locxia is a data platform focused on housing data. Developed by Ntico's data experts, it empowers businesses and local authorities to better understand their customers and territories. The platform provides access to over 250 value-added data sourced from satellite images and public data; all consolidated and accessible on a dedicated platform.
```

File: content/blogs/2023-06-21-locxia-kestra.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+They provides a wide range of data at a very low granularity:
+
+* Outdoor development: presence of gardens and swimming pools, surfaces, etc.
+* Buildings & installations: nature, ground surfaces, etc.
+* Energy diagnosis: Energy Performance Diagnostics (DPE), etc.
+
+
+## Kestra
+
+To support high data volumetry and scalability requirements, Ntico experts choose Kestra as the data orchestrator behind Locxia.
```
Comment (anna-geller) [inline content/blogs/2023-06-21-locxia-kestra.md:45]:
```suggestion
To support high data volume and scalability requirements, Ntico experts chose Kestra as the data orchestrator powering Locxia.
```

File: content/blogs/2023-06-21-locxia-kestra.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+
+> “Kestra's scalable and cost-effective data orchestration capabilities provided the foundation for our data platform Loxcia, allowing us to effectively handle and process vast amounts of geospatial data.” Emmanuel Santerne - NTICO Associate
+
+
+## Conclusion
+
+The collaboration between Ntico and Kestra has been instrumental in the development of Locxia, a modern data platform that revolutionizes the understanding of housing data. As specialized in logistics optimization and data-driven solutions, Ntico recognized the need for a robust data orchestrator to handle the vast volumes of geospatial data involved in Locxia.
+
+By leveraging Kestra's advanced data ingestion and processing capabilities, Ntico successfully tackled the challenges associated with diverse data sources, including APIs, open-data platforms, and raw satellite images. Kestra served as the backbone of Locxia, effectively coordinating data ingestion processes.
```
Comment (anna-geller) [inline content/blogs/2023-06-21-locxia-kestra.md:81]:
```suggestion
By leveraging Kestra's simple data ingestion and processing capabilities, Ntico successfully tackled the challenges associated with diverse data sources, including APIs, open-data platforms, and raw satellite images. Kestra served as the backbone of Locxia, effectively coordinating data ingestion processes.
```

**Review by anna-geller — CHANGES_REQUESTED** [review]
nice! 🚀

File: content/blogs/2023-06-21-locxia-kestra.md (hunk 1 above)
Comment (Ben8t) [reply content/blogs/2023-06-21-locxia-kestra.md]: I keep the active tense for the first sentence; but applied the rest ;)

**Review by anna-geller — APPROVED** [review]

**Pull request merged**